src_new/bidManager.js

```diff
@@ -339,7 +339,8 @@ exports.executeAnalyticsPixel = function(){ // TDD, i/o : done
 		if(slots.length > 0){
 			outputObj.s = slots;
 			outputObj[CONSTANTS.COMMON.IMPRESSION_ID] = window.encodeURIComponent(impressionID);
-			(new window.Image()).src = pixelURL + "&json=" + window.encodeURIComponent(JSON.stringify(outputObj));
+			// (new window.Image()).src = pixelURL + "&json=" + window.encodeURIComponent(JSON.stringify(outputObj));
```

Comment:
1. keep arguments to a function separated by space after a comma
2. I think you will need to pass the last argument to the function util.utilAjaxCalll as "json=" + JSON.stringify(outputObj)
3. Check with Isha before giving QA drop.